request.js

```diff
@@ -1411,6 +1405,19 @@ Request.prototype.destroy = function () {
     self.response.destroy()
   }
 }
+Request.prototype.enableUnixSocket = function () {
```

Comment:
I've fixed all the things you highlighted, but I can make it a seperate by doing `fn (uri, context){}` or something if you prefer.